src/agenda/index.js

```diff
@@ -123,12 +125,27 @@ export default class AgendaView extends Component {
     }
   };
 
-  enableCalendarScrolling() {
+  /**
+   * Opens/closes the calendar programmatically.
+   * @param {bool} open `true` when calendar should open. Otherwise `false`.
+   */
+  setCalendarOpen = (open) => {
+    const maxY = this.initialScrollPadPosition();
+      if (open) {
+        this.setScrollPadPosition(0, true);
+        this.enableCalendarScrolling(true);
+      } else {
+        this.setScrollPadPosition(maxY, true);
+        this.enableCalendarScrolling(false);
+      }
```

Comment:
You're calling the same methods for both positions, only the param change. so why not do something like: `this.setScrollPadPosition(open  ? 0 : maxY, true);
this.enableCalendarScrolling(open);`

Reply:
Updated!